src/ui/AP2DataPlot2D.cpp

```diff
@@ -1074,7 +1053,7 @@ void AP2DataPlot2D::itemEnabled(QString name)
         }
         // use golden ratio for evenly distributed colors
         double golden_ratio_conjugate = 0.618033988749895;
-        double h = ((double)rand() / (double)(RAND_MAX));
+        double h = (static_cast<double>(rand()) / static_cast<double>(RAND_MAX));
```

Comment:
typically you only need to cast one side to a double to force floating point division. if static cast look long winded for typing double(rand())/RAND_MAX is my preferred style.